test/mocha.opts

```diff
@@ -0,0 +1,3 @@
+--require ./node_modules/ts-node/register
+--recursive
+test/**/*.ts
```

Comment:
You said this was autogenerated, right?

Reply:
We should remove this file. This used to be the way to configure mocha but now it's deprecated in favor of a `.mocharc.js` in the root of the repository, which we have.